crowbar_framework/app/views/nodes/_form.html.haml

```diff
@@ -113,6 +113,25 @@
           %span.help-block
             = t('.availability_zone_hint')
 
+
+      .form-group
+      = f.label :disks_roles, t('.disks_roles')
+
+      %table.table.table-middle
+        %thead
+          %tr
+            %th.col-sm-3
+              = t(".disk_name")
+            %th.col-sm-2
+              = t(".disk_role")
+        %tbody
+          - @node.disks_roles.each do |disk_name, d|
+            %tr
+              %td
+                = text_field_tag "disks_roles[#{disk_name}][name]", disk_name, :class => "form-control"
+              %td
+                = text_field_tag "disks_roles[#{disk_name}][role]", d["role"], :placeholder => t(".disk_role_undecided"), :class => "form-control"
```

Comment:
I'd feel more comfortable if that'd be a menu tag so that people can't enter something wrong. Would require a way for barclamps to say they can claim disks, though.